stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -1234,11 +1234,18 @@ llvm::Optional<TypeLookupError> swift::_checkGenericRequirements(
     llvm::ArrayRef<GenericRequirementDescriptor> requirements,
     llvm::SmallVectorImpl<const void *> &extraArguments,
     SubstGenericParameterFn substGenericParam,
-    SubstDependentWitnessTableFn substWitnessTable) {
+    SubstDependentWitnessTableFn substWitnessTable,
+    bool skipUnknownKinds) {
   for (const auto &req : requirements) {
     // Make sure we understand the requirement we're dealing with.
-    if (!req.hasKnownKind())
-      return TypeLookupError("unknown kind");
+    if (!req.hasKnownKind()) {
```

Comment:
so the kind here ends up being some weird 24 for the Sendable requirement...
Since it has an unknown kind we can't really get the mangled name or protocol etc to check it in detail if it is the exact sendable.

But the same issue would happen with any other @_marker protocol I suppose...